native/src/widget/pick_list.rs

```diff
@@ -114,6 +116,12 @@ where
         self
     }
 
+    /// Sets the message sent when [`PickList`] selection changes
+    pub fn on_change(mut self, msg: Message) -> Self {
```

Comment:
Why is the `on_change` trigger necessary? Shouldn't we be able to use the `on_selected` one instead? It seems a bit redundant.

Reply:
Yep totally redundant. It's now removed. Thanks!